redisinsight/api/provider-interfaces/models/command-execution.ts

```diff
@@ -0,0 +1,20 @@
+// todo: discuss. introduce userId here?
+export class CommandExecution {
+  id: string;
+
+  databaseId: string;
+
+  command: string;
```

Comment:
We need to discuss security aspects of accepting the command.

Unlike in the on-premise scenario, in the cloud scenario we need to be very careful about receiving the command and running it as-is...

I am assuming right now we pass the command as-is. Is that correct ? 

(Definitely we need to discuss in depth, but as a starter, I think it needs to be encoded (URL or even Base64 ?))